environments/nodejs/Dockerfile-12.16

```diff
@@ -0,0 +1,18 @@
+# A docker image for the func container.
+
+# default variant is the official alpine node image (much smaller than the standard image)
```

Comment:
Not super relevant comments, let's remove these.

Reply:
Done